java/app-encryption/src/main/java/com/godaddy/asherah/appencryption/persistence/InMemoryMetastoreImpl.java

```diff
@@ -41,4 +41,9 @@ public boolean store(final String keyId, final Instant created, final T value) {
       return true;
     }
   }
+
+  @Override
+  public String getRegionSuffix() {
+    return "";
+  }
```

Comment:
can add this default implementation to the `Metastore` interface. Would help in reducing a lot of redundant code.

Reply:
Having separate implementations is better in terms of extensibility if at  some point we decide to enable region suffix for the JDBC version. That being said, I am ok changing this to a `default` method if you feel too strongly about  it.

Reply:
discussed about this, we can override method as/when required in JDBC version.